app/state_test.go

```diff
@@ -1148,7 +1148,7 @@ type stateTransitionsWithUpdateModulesTestCase struct {
 
 var stateTransitionsWithUpdateModulesTestCases []stateTransitionsWithUpdateModulesTestCase = []stateTransitionsWithUpdateModulesTestCase{
 
-	stateTransitionsWithUpdateModulesTestCase{
+	{
```

Comment:
Yes!

But just FYI, IIRC I originally coded it like this, but had to add the explicit types due to old Golang versions in Yocto. That was a while back though, if dunfell is ok with this, then we are good to go.

Reply:
I usually run `go fmt` with the `-s` option. The only time you have to use explicit struct declarations are for slices of interfaces as far as I know.